projects/storefrontlib/shared/components/assistive-technology-message/assistive-technology-message.directive.ts

```diff
@@ -0,0 +1,48 @@
+import {
+  Directive,
+  ElementRef,
+  HostListener,
+  Input,
+  Optional,
+  TemplateRef,
+} from '@angular/core';
+import { GlobalMessageService } from '../../../../core/src/global-message/facade/global-message.service';
+import { GlobalMessageType } from '../../../../core/src/global-message/models/global-message.model';
+
+@Directive({
+  selector: '[cxAtMessage]',
+})
+export class AtMessageDirective {
+  /**
+   * Usage [cxAtMEssage]="'translatableKey' | cxTranslate"
+   */
+  @Input() cxAtMessage: string;
```

Comment:
may be just input "translatableKey"? This key can be passed to `this.globalMessageService.add`.

Reply:
Yes this is an option as well, but since we have to considers translatable keys that have parameters (I.e. 
<img width="529" alt="Screen Shot 2022-02-02 at 9 47 51 AM" src="https://user-images.githubusercontent.com/49131189/152176818-e0aeb7a2-e4d7-4968-80fe-7fcc86d79cc3.png">

that means that the input of the directive would have to look like this:
<img width="452" alt="Screen Shot 2022-02-02 at 9 48 39 AM" src="https://user-images.githubusercontent.com/49131189/152176968-8b1e87be-77db-41fc-b8ed-7632e9319ebe.png">

Whereas I feel that translating first in template and passing translated string is simpler and uniform with the rest of the application:
<img width="436" alt="Screen Shot 2022-02-02 at 9 51 28 AM" src="https://user-images.githubusercontent.com/49131189/152177466-dd26d29f-0031-48c5-81da-5a0a8b84f719.png">